WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js

```diff
@@ -1831,6 +1831,7 @@ ZmMailMsgView.prototype._renderMessageBody1 = function(params, part) {
         // calendar part in ICS format
         else if (ct === ZmMimeTable.TEXT_CAL) {
             content = ZmMailMsg.getTextFromCalendarPart(bp);
+            content = AjxStringUtil.htmlEncode(content);
```

Comment:
Short answer, probably not.  Longer answer, maybe but it depends on the documented behavior and actual use cases.  In general, it's ok for the data to be returned in a "pristine" form, but the caller needs to be sure that the data gets encoded properly for the context where that data is used.

Given the name getText... I suspect that we shouldn't change that function.